core/controllers/editor.py

```diff
@@ -32,7 +31,7 @@
 from core.domain import exp_services
 from core.domain import fs_domain
 from core.domain import fs_services
-from core.domain import html_validation_service
+from core.domain import image_validation_service
```

Comment:
image_validation_service**s*** considering this is a module and we are expecting to have more methods in this file [Also, to have a consistent filename.]

Reply:
Done